code/drasil-example/swhs/lib/Drasil/SWHS/GenDefs.hs

```diff
@@ -109,10 +111,11 @@ rocTempDerivArbVol :: [Sentence]
 rocTempDerivArbVol = [S "We consider an arbitrary" +:+. phrase vol,
   atStartNP (the volHtGen), S "is assumed constant. Then", eqN 1, S "can be written as"]
 
-rocTempDerivConsFlx :: Sentence -> [ConceptInstance] -> [Sentence]
-rocTempDerivConsFlx s assumps = [S "Where", 
+rocTempDerivConsFlx :: (HasUID r, HasRefAddress r, HasShortName r) => r
+ -> Sentence -> [ConceptInstance] -> [Sentence]
+rocTempDerivConsFlx r s assumps = [S "Where", 
   foldlList Comma List (map ch [htFluxIn, htFluxOut, inSA, outSA]),
-  S "are explained in" +:+. refS rocTempSimp, s, S "Assuming", 
+  S "are explained in" +:+. refS r, s, S "Assuming",  -- FIXME: refS r is a hack to share the rocTempSimpRC instance without causing friction between nopcm and swhs
```

Comment:
This `rocTempDerivConsFlx` reference expected the variant of `rocTempSimp` belonging to SWHS, but it was accidentally referenced in nopcm. So, when I changed the UIDs for the `rocTempSimp` in nopcm, this reference broke since the `rocTempSimp` of SWHS wasn't registered in NoPCM's ChunkDB. In other words, this was a reference bug that happened to not cause problems because the UIDs lined up unintentionally.

Reply:
I'm not sure why this was posted twice.